utils/test-cleanup.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+
+if [ "$(whoami)" != "root" ] ; then
+    sudo "$0"
+    exit 0
+fi
+
+echo 0 > /proc/sys/net/ipv4/ip_forward
```

Comment:
This is for testing on people's laptops, not on actual ATC gateways. I think it's reasonable to assume you aren't setting up your laptop as a network gateway.